easybuild/easyconfigs/b/bibtexparser/bibtexparser-0.5-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'PythonPackage'
+
+name = 'bibtexparser'
+version = '0.5'
```

Comment:
0.6.0 is the most recent one, but I need 0.5 as a dependency for something

I didn't invent this crazy versioning scheme; 0.5.0 doesn't exist :)